src/main/core/Event/CatalogEvents/ResourceEvents.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+namespace Claroline\CoreBundle\Event\CatalogEvents;
+
+final class ResourceEvents
+{
+    /**
+     * @Event("Claroline\CoreBundle\Event\Resource\LoadResourceEvent")
+     */
+    public const RESOURCE_OPEN = 'resource.load';
```

Comment:
`resource_load` for consistency?

Reply:
Yes totally agree, renaming done, thx :)